.github/actions/setup-solana-anchor/action.yml

```diff
@@ -0,0 +1,22 @@
+name: Setup Solana and Anchor
+inputs:
+  verify_version:
+    description: "Solana verify version to install (leave empty to skip)"
+    required: false
+    default: ""
+  node_version:
+    description: "Node version to install"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Extract Solana versions
```

Comment:
Nit: think we could reuse this in the publish workflow too?

Reply:
yes, indeed - I planned handling publish workflow in a follow-up